<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1800</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/res/iconWindow.jpg</normaloff>:/res/iconWindow.jpg</iconset>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuProject">
    <property name="title">
     <string>工程</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionClose"/>
    <addaction name="separator"/>
    <addaction name="actionLoad"/>
    <addaction name="actionExport"/>
    <addaction name="actionSave"/>
    <addaction name="actionLatestUse"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>视图</string>
    </property>
    <addaction name="actionLargestView"/>
    <addaction name="actionBigView"/>
    <addaction name="actionSmallView"/>
    <addaction name="separator"/>
    <addaction name="actionListView"/>
    <addaction name="actionDetailedView"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="actionRun"/>
    <addaction name="actionDeplovement"/>
    <addaction name="actionRemake"/>
    <addaction name="actionPackUp"/>
   </widget>
   <addaction name="menuProject"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="separator"/>
   <addaction name="actionExit"/>
   <addaction name="actionRemake"/>
   <addaction name="actionPackUp"/>
   <addaction name="separator"/>
   <addaction name="actionLargestView"/>
   <addaction name="separator"/>
  </widget>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/res/iconNew.jpg</normaloff>:/res/iconNew.jpg</iconset>
   </property>
   <property name="text">
    <string>新建工程</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/res/iconOpen.jpg</normaloff>:/res/iconOpen.jpg</iconset>
   </property>
   <property name="text">
    <string>打开工程</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>关闭工程</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="text">
    <string>导入</string>
   </property>
  </action>
  <action name="actionExport">
   <property name="text">
    <string>导出</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>保存</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionLatestUse">
   <property name="text">
    <string>最近使用文件</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/res/EXIT.jpg</normaloff>:/res/EXIT.jpg</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="actionLargestView">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/res/other.jpg</normaloff>:/res/other.jpg</iconset>
   </property>
   <property name="text">
    <string>最大</string>
   </property>
  </action>
  <action name="actionBigView">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/res/other.jpg</normaloff>:/res/other.jpg</iconset>
   </property>
   <property name="text">
    <string>大图</string>
   </property>
  </action>
  <action name="actionSmallView">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/res/other.jpg</normaloff>:/res/other.jpg</iconset>
   </property>
   <property name="text">
    <string>小图</string>
   </property>
  </action>
  <action name="actionListView">
   <property name="text">
    <string>列表</string>
   </property>
  </action>
  <action name="actionDetailedView">
   <property name="text">
    <string>详细</string>
   </property>
  </action>
  <action name="actionRun">
   <property name="text">
    <string>运行</string>
   </property>
  </action>
  <action name="actionDeplovement">
   <property name="text">
    <string>开发</string>
   </property>
  </action>
  <action name="actionRemake">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/res/remake.jpg</normaloff>:/res/remake.jpg</iconset>
   </property>
   <property name="text">
    <string>恢复</string>
   </property>
  </action>
  <action name="actionPackUp">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/res/packUp.jpg</normaloff>:/res/packUp.jpg</iconset>
   </property>
   <property name="text">
    <string>打包</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
